bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java

```diff
@@ -325,6 +328,26 @@ static void addPrePushComment(PullRequest pr, Hash hash, String extraMessage) {
         pr.addComment(commentBody.toString());
     }
 
+    static void processBackportLabel(PullRequest pr, List<Comment> allComments) {
+        var botUser = pr.repository().forge().currentUser();
+        for (String label : pr.labelNames()) {
+            var matcher = BACKPORT_LABEL_PATTERN.matcher(label);
+            if (matcher.matches()) {
+                var repoName = matcher.group(1);
+                var branchName = matcher.group(2);
+                var text = "Creating backport for repo " + repoName + " on branch " + branchName
+                        + "\n<!--\n /backport " + repoName + " " + branchName + "\n-->" + "\n"
+                        + PullRequestCommandWorkItem.VALID_BOT_COMMAND_MARKER;
```

Comment:
In `CheckWorkItem`, when adding `/integrate` based on an earlier `/integrate auto`, we don't hide the command. I think we should be consistent and not hide it here either.

Reply:
Sure